src/bptl/work_units/zac/tasks.py

```diff
@@ -0,0 +1,70 @@
+import json
+
+from requests import Response
+
+from bptl.tasks.base import WorkUnit, check_variable
+from bptl.tasks.registry import register
+
+from .client import ZACClient
+from .serializers import ZacUsersDetailsSerializer
+
+
+@register
+class UserDetailsTask(WorkUnit):
+    """
+    Requests email and name data from usernames from the zac.
+
+    In the camunda process models accorderen/adviseren we have a list of usernames from
+    the zac. In order to send signalling emails, we will need to fetch
+    the email addresses and names from the zac and feed it back to the camunda process.
+
+    In this first implementation a simple and direct get request is done
+    at the zac.accounts.api endpoint.
+
+    The task requires the service with alias "zac" to be configured in the work_unit
+    services and be registered in the ZAC where the token will be set.
```

Comment:
Currently not the case because no `zgw_consumers.Service` is used, rather it's using the static configuration.